<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupWgt</class>
 <widget class="QWidget" name="CreateGroupWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>群聊名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170,168,159);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget#tableWidget {
	background-color:transparent;
	font: 25 12pt '微软雅黑 Light';
	color: rgb(31,31,31);
}</string>
     </property>
     <column>
      <property name="text">
       <string>复选框</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>头像</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>昵称</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border-radius: 15px;
}

    QPushButton:hover {
        background-color: rgb(170,168,159);
    }

    QPushButton:pressed {
        background-color: rgb(115,110,95);
    }</string>
       </property>
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
